### PR TITLE
Move everything over to RHEL8

### PR DESCRIPTION
**Pull request by jesse-dds**
https://ccpo.atlassian.net/browse/AT-5343

Convert our ATAT Docker image to use a hardened version of RHEL8.

### CODE REVIEW

File: python.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+FROM cloudzeropwdevregistry.azurecr.io/rhelubi:8.2
```
Comment (dandds) [inline python.Dockerfile:1]: Likewise for the base image here.

File: Dockerfile [hunk 2]
```diff
@@ -62,24 +69,50 @@ ENV GIT_SHA "${GIT_SHA}"
...
-RUN apk update && \
-      apk --no-cache add \
-      dumb-init \
-      postgresql-client \
-      postgresql-dev \
-      postgresql-libs \
-      uwsgi-logfile \
-      uwsgi-python3
+# TODO(heyzoos): Make this all work as part of the first stage.
+# Get the latest GPG key for enterprise linux 8.
```
[the remark this answers is not in the document]
Comment (jesse-dds) [reply Dockerfile:94]: https://ccpo.atlassian.net/browse/AT-5381

**Comment by carleton-dds** [comment]
What's the purpose of the `python.Dockerfile` file?

**Comment by jesse-dds** [comment]
> What's the purpose of the `python.Dockerfile` file?

It is a version of the RHEL image with python 3.7.3 exactly. Saves us from having to compile python from source.

**Pull request merged**